src/editors/index.ts

```diff
@@ -42,6 +42,25 @@ export class DebuggerEditors extends TabPanel {
     this.editorFactory = options.editorServices.factoryService.newInlineEditor;
     this.mimeTypeService = options.editorServices.mimeTypeService;
 
+    this.debuggerService.session?.clientChanged.connect((_, updated) => {
+      if (this.tabBar?.currentTitle?.owner?.id === updated.path) {
```

Comment:
We probably don't need to use the optional chaining with `?` everywhere.

For example here `this.tabBar` should not be `null`.

Reply:
tabBar exist before. I will just check if currentTitle exist.

Reply:
for session i put optional chaining cause test are not passed

Reply:
Due to we want always to have the possibility to set breakpoints even for not focused editors I removed a connection to clienteChanged and tabBar changed.

Reply:
Then do we need to keep the `clientChanged` signal?

Reply:
I removed that

Reply:
:+1: